app/src/auth/components/Session/index.jsx

```diff
@@ -0,0 +1,61 @@
+// @flow
+
+import React, { type Node } from 'react'
+
+import Context, { type Props as State } from '$auth/contexts/Session'
+import { isLocalStorageAvailable } from '$mp/utils/storage'
+
+const SESSION_TOKEN_KEY = 'session.token'
+
+type Props = {
+    children: Node,
+}
+
+class Session extends React.Component<Props, State> {
+    static token(): ?string {
+        if (isLocalStorageAvailable()) {
+            return localStorage.getItem(SESSION_TOKEN_KEY) || null
```

Comment:
Maybe session storage is better to store the login, that is cleared when the browser/window closes.

Reply:
Do you think it's more desirable to log users out when they close their browsers? I can roll with that, just please confirm.

Reply:
The data stored in `localStorage` persists until explicitly deleted - for me it would make sense that the session token (which has an expiration) can be cleared when the window or browser is closed.

Reply:
Will change! 😃